pyart/graph/radardisplay.py

```diff
@@ -1019,6 +1049,29 @@ def _label_axes_vpt(self, axis_labels, ax):
         else:
             ax.set_ylabel(y_label)
 
+    def _set_vpt_time_axis(self, ax, date_time_form=None, tz=None):
+        """ Set the x axis as a time formatted axis.
+
+        Parameters
+        ----------
+        ax : Matplotlib axis instance
+            Axis to plot. None will use the current axis.
+        date_time_form : str
+            Format of the time string for x-axis labels.
+        tz : str
+            Time zone info to use when creating axis labels (see datetime).
+        """
+        if date_time_form is None:
+            date_time_form = '%H:%M'
+        if date_MinTicker is None:
```

Comment:
date_MinTicker is not defined in this scope, I believe this if statement can be removed.